### PR TITLE
DEVELOPER-2926 Add link to CDK 2.0 Release Notes to CDK pages

### PR DESCRIPTION
**Pull request by paulrobinson**

### CODE REVIEW

**Comment by RubyGemTSL** [comment]
retest this please

**Comment by LightGuard** [comment]
retest this please

File: products/cdk/_common/product.yml [hunk 1]
```diff
@@ -34,6 +34,16 @@ guides:
         url: https://access.redhat.com/webassets/avalon/d/Red_Hat_Enterprise_Linux_Atomic_Host-7-Getting_Started_with_Container_Development_Kit-en-US/Red_Hat_Enterprise_Linux_Atomic_Host-7-Getting_Started_with_Container_Development_Kit-en-US.pdf
       epub:
         url: https://access.redhat.com/webassets/avalon/d/Red_Hat_Enterprise_Linux_Atomic_Host-7-Getting_Started_with_Container_Development_Kit-en-US/Red_Hat_Enterprise_Linux_Atomic_Host-7-Getting_Started_with_Container_Development_Kit-en-US.epub
+  CDK_2_Release_notes
```
Comment (LightGuard) [inline products/cdk/_common/product.yml]: You forgot the `:` at the end of this line

**Comment by paulrobinson** [comment]
Robert Kratky has approved from a 'requestor' perspective.

Please do a code review.

**Comment by LightGuard** [comment]
Looks good to me.

**Pull request merged**